scripts/validationAndPreparation.sh

```diff
@@ -20,19 +22,18 @@ echo "files are $FILES"
 
 unset NEWEST_CONTRIBUTION
 
-#NEWEST_CONTRIBUTION="$(find_newer_contribution $DATE_OF_NEWEST_CONTRIBUTION)"
+#searching for newest contribution 
 echo "search for files newer than ${DATE_OF_NEWEST_CONTRIBUTION}"
 for f in $FILES
```

Comment:
Is this really files? Isn't this the top level folders?

Reply:
good call. I will rename it